src/operations/update.ts

```diff
@@ -69,4 +74,54 @@ export class UpdateOperation implements Operation<UpdateOperationInput> {
 
     return JSON.parse(content);
   }
-}
+
+  private async buildExpression(updateExp: string, input: UpdateOperationInput) {
+    const updateExpressions = input.updateExpression.split(',');
+
+    for(let i=0; i<updateExpressions.length; i++) {
+      if(i===0 && updateExpressions.length > 1) {
+        updateExp = 'set'.concat(` ${updateExpressions[i]} = :${updateExpressions[i]},`);
```

Comment:
this is look odd its like mixing programming language conventions.
```
'set'.concat(` ${updateExpressions[i]} = :${updateExpressions[i]},`);
```
is equivelent to
```
`set ${updateExpressions[i]} = :${updateExpressions[i]},`
```
unless I'm mistaken